applications/mne_anonymize/mne_anonymize.pro

```diff
@@ -123,7 +123,7 @@ HEADERS += \
 FORMS += \
     mainwindow.ui
 
-*-clang++ {
+clang {
```

Comment:
why is this needed in only this one file? Weren't other files changed as well?

Reply:
all other 109 appearances where changed in #872 . This one escaped.